extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -580,6 +584,7 @@ describes.realWin('Events', {amp: 1}, env => {
               target,
               matchEmptySpec,
               readyPromise,
+              /* createReadyReportPromiseFunc */ null,
```

Comment:
We also need tests to make sure we add calls with non-null `createReadyReportPromiseFunc`?